css/doc.scss

```diff
@@ -32,3 +32,25 @@ permalink: /css/doc.css
 		padding: 2px 7px 0;
 	}
 }
+
+@media screen and (max-width: 480px) {
+	body {
+		// prevent background leak
+		background: #eee;
```

Comment:
Shouldn't this apply to the rest of the website as well? Additionally, I'm not sure about overriding the background color like this and not either
a) using a variable to avoid duplicating the background color, or
b) using `unset` (?) on the `background-image` property (if possible).

Reply:
What do you mean by "the rest of the website"? Like in general, disregarding the device it's viewed on? Then the background image would vanish completely and we could be thinking of removing its definition entirely.
I put it only for phones because the background-image is never visible there anyways (it only shows up on screen larger then tablets).

The `background: unset` would have to happen on the html element, because the definition for the background is on it. An `unset` would lead to a white background though! So we'd have to go with choice _a)_.

Reply:
I mean applying this style rule to the whole website instead of just the docs.

Reply:
Oh, now i see. Sorry. Ok, will do.